r2/r2/controllers/front.py

```diff
@@ -514,7 +514,7 @@ def GET_newreddit(self, name):
 
     @pagecache_policy(PAGECACHE_POLICY.LOGGEDIN_AND_LOGGEDOUT)
     @require_oauth2_scope("modconfig")
-    @api_doc(api_section.moderation, uses_site=True)
+    @api_doc(api_section.subreddits, uses_site=True)
```

Comment:
@xiongchiamiov the reason I made this change, is because you don't need to be a mod to view the raw css stylesheet (nor the actual /about/stylesheet that is returned in `GET_editreddit` which is why I added the uri variants there). That said, maybe the scope should be changed too?